src/ert/config/gen_kw_config.py

```diff
@@ -342,7 +342,7 @@ def _parse_transfer_function(param_string: str) -> TransferFunction:
             "NORMAL": ["MEAN", "STD"],
             "LOGNORMAL": ["MEAN", "STD"],
             "TRUNCATED_NORMAL": ["MEAN", "STD", "MIN", "MAX"],
-            "TRIANGULAR": ["XMIN", "XMODE", "XMAX"],
```

Comment:
in this file there is a function called `trans_triangular`, which still uses variables `_xmin, _xmax, _xmode`. Should we rename those too for the naming consistency?

Reply:
Sure 👍